modules/contact-form/js/gutenblock.jsx

```diff
@@ -0,0 +1,630 @@
+			<GrunionFieldSettings
+				required={ props.required }
+				setAttributes={ props.setAttributes }
+			/>
+			<div className="grunion-field">
+				<TextareaControl
+					label={ <GrunionFieldLabel
+						required={ props.required }
+						label={ props.label }
+						setAttributes={ props.setAttributes }
+					/> }
+					disabled={ true }
+				/>
+			</div>
+		</Fragment>
+	);
+}
+
+function GrunionFieldCheckbox( props ) {
+	return (
+		<Fragment>
+			<GrunionFieldSettings
+				required={ props.required }
+				setAttributes={ props.setAttributes }
+			/>
+			<div className="grunion-field">
+				<CheckboxControl
+					label={ <GrunionFieldLabel
+						required={ props.required }
+						label={ props.label }
+						setAttributes={ props.setAttributes }
+					/> }
+					disabled={ true }
+				/>
+			</div>
+		</Fragment>
+	);
+}
+
+class GrunionFieldMultiple extends Component {
+	constructor( ...args ) {
+		super( ...args );
+		this.onChangeOption = this.onChangeOption.bind( this );
+	}
+
+	onChangeOption( key = null, option = null ) {
+		const newOptions = this.props.options.slice( 0 );
+		if ( 'object' === typeof key ) {
+			newOptions.push( '' );
+		} else if ( null === option ) {
+			newOptions.splice( key, 1 );
+		} else {
+			newOptions.splice( key, 1, option );
+		}
+		this.props.setAttributes( { options: newOptions } );
+	}
+
+	render() {
+		return (
+			<Fragment>
+				<GrunionFieldSettings
+					required={ this.props.required }
+					setAttributes={ this.props.setAttributes }
+				/>
+				<div className="grunion-field">
+					<GrunionFieldLabel
+						required={ this.props.required }
+						label={ this.props.label }
+						setAttributes={ this.props.setAttributes }
+					/>
+					<ol>
+						{ this.props.options.map( ( option, index )=>(
+							<GrunionOption
+								key={ index }
+								option={ option }
+								index={ index }
+								onChangeOption={ this.onChangeOption }
+							/>
+						) ) }
+					</ol>
+					<IconButton
+						icon="insert"
+						label={ __( 'Insert option' ) }
+						onClick={ this.onChangeOption }
+					> { __( 'Add' ) }</IconButton>
+				</div>
+			</Fragment>
+		)
+	}
+}
+
+class GrunionOption extends Component {
+	constructor( ...args ) {
+		super( ...args );
+		this.onChangeOption = this.onChangeOption.bind( this );
+		this.onDeleteOption = this.onDeleteOption.bind( this );
+	}
+
+	onChangeOption( x ) {
+		this.props.onChangeOption( this.props.index, x.target.value );
+	}
+
+	onDeleteOption() {
+		this.props.onChangeOption( this.props.index );
+	}
+
+	render() {
+		return (
+			<li>
+				<input
+					type="text"
+					className="option"
+					value={ this.props.option }
+					placeholder={ __( 'Enter your option value here…' ) }
+					onChange={ this.onChangeOption }
+				/>
+				<IconButton
+					icon="no"
+					label={ __( 'Remove option' ) }
+					onClick={ this.onDeleteOption }
+				/>
+			</li>
+		);
+	}
+}
+
+/**
+ * Block Registrations:
+ */
+
+registerBlockType( 'grunion/form', {
```

Comment:
Let's stick to the same namespace as the rest of the Jetpack blocks use? `a8c/form` 

Another discussion if it should be `jetpack/form` instead but whatever the name, consistency is the way to go. 👌

Reply:
Yarp, I need to do a pass renamespacing it soon.  Grunion was just the legacy handle of the module.